pkg/server/app/fake_test.go

```diff
@@ -182,6 +182,30 @@ func TestFakeOperationsList(t *testing.T) {
 	}
 }
 
+func TestFakeOperationsListByTeam(t *testing.T) {
+	var testCases = []struct {
+		app      *App
+		teamName string
+		expected int
+	}{
+		{&App{Name: "teresa", Team: "gophers"}, "gophers", 1},
+		{&App{Name: "teresa", Team: "vimmers"}, "gophers", 0},
+	}
+
+	fake := NewFakeOperations()
+
+	for _, tc := range testCases {
+		fake.(*FakeOperations).Storage[tc.app.Name] = tc.app
+		apps, err := fake.ListByTeam(tc.teamName)
+		if err != nil {
+			t.Fatal("error getting app list by team:", err)
```

Comment:
Space after team

Reply:
Actually isn't necessary, go appends a space after the main string in non-f (printf vs print) functions.